### ISSUE TITLE
Customizing menu entries

### ISSUE THREAD

**Issue by SystematicError** [ms-jpq/coq_nvim#455]
Is the following possible to change individual icon highlights and icon positioning in a menu entry? Currently I have it looks like this:

![image](https://user-images.githubusercontent.com/63366086/152652406-902001d7-b70c-41a7-889f-898c607817c7.png)

This is how I would like for my final menu entry order to look like:

```
icon code source
```

It would also be nice to have the ability to highlight these, such as different colors for each icon and a darker color for the source.

Based on what I've seen in the [docs](https://github.com/ms-jpq/coq_nvim/blob/coq/docs/DISPLAY.md), this doesn't currently seem possible, am I missing something or does this have to be implemented?

**Comment by kaiuri** [comment]
@SystematicError  yes you can, just add your custom colors to a syntax file appending to COQ's own syntax file

**Comment by kaiuri** [comment]
about the positining, thou, that would require messing up with how the sources dictionary is displayed... that's beyond me, i suck at python.

**Comment by SystematicError** [comment]
Ah thanks, unfortunately I am in the process of moving from nvim to another text editor, however if you do get any more info please do post a follow up. Thanks!

**Comment by kaiuri** [comment]
@SystematicError you should consider [nvim-cmp](https://github.com/hrsh7th/nvim-cmp/) if you really want those colors and layouts. Since it's written in lua, it's VERY easy to customize it the way you want.
> Ah thanks, unfortunately I am in the process of moving from nvim to another text editor, however if you do get any more info please do post a follow up. Thanks!

**Comment by luisiacc** [comment]
@kaiuri I'm interested on what you meant by this:
> @SystematicError yes you can, just add your custom colors to a syntax file appending to COQ's own syntax file
